nfts/frontend/package.json

```diff
@@ -6,16 +6,23 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.2.1",
```

Comment:
npm install showing vunlerabilities

npm install


up to date, audited 1457 packages in 3s

214 packages are looking for funding
  run `npm fund` for details

6 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Reply:
fixed